src/snowplow/EventHandler.ts

```diff
@@ -0,0 +1,34 @@
+import { PayloadBuilder, SelfDescribingJson } from '@snowplow/tracker-core';
+import * as Sentry from '@sentry/node';
+import { Tracker } from '@snowplow/node-tracker';
+
+export class EventHandler {
+  constructor(protected tracker: Tracker) {
+    this.tracker = tracker;
+  }
+
+  /**
+   * Track snowplow event
+   * @param event
+   * @param context
+   * @private
+   */
+  protected async track(
+    event: PayloadBuilder,
+    context: SelfDescribingJson[]
+  ): Promise<void> {
+    try {
+      await this.tracker.track(event, context);
```

Comment:
noting again that the snowplow node library is not async. i don't think having `await` here will cause a problem, but reading the code you may get the wrong idea.

this whole function probably doesn't need to be `async` unless/until snowplow updates their library to be async.